pages/mint-nft/index.vue

```diff
@@ -474,6 +478,18 @@ watch(iscnData, (recordData) => {
   }
 })
 
+const validate = (state: any): FormError[] => {
+  hasError.value = false
+  const errors = []
+  const whitespaceRegex = /\s/
```

Comment:
Please use `[a-zA-Z][a-zA-Z0-9/:-]{2,100}` to check valid nft id

Reply:
[ef672ba](https://github.com/likecoin/nft-book-press/pull/84/commits/ef672ba0a7e2323d777a7345452dd93f3b6185dd)